kythe/typescript/kythe.ts

```diff
@@ -110,7 +110,6 @@ export function makeOrdinalEdge(edge: EdgeKind, ordinal: number): OrdinalEdge {
  *   https://github.com/kythe/kythe/tree/master/kythe/data/schema_index.textproto#L64
  */
 export enum NodeKind {
-  ABSVAR = 'absvar',
```

Comment:
This was already out of sync with the missing ABS enum. Maybe there should be two "UNUSED_1" and "UNUSED_2" in there?

Reply:
Do these actually have to match the proto enum values?  It seems more like the C++ enum where the integers don't matter.

Reply:
Everything seems to work, and it already didn't match, so I'm assuming it doesn't matter.